upgrading/manual_upgrades.adoc

```diff
@@ -416,7 +416,7 @@ updated.
 == Updating Policy Definitions
 
 After a cluster upgrade, the default roles
-xref:../../architecture/additional_concepts/authorization.adoc#roles[default
+xref:../architecture/additional_concepts/authorization.adoc#roles[default
```

Comment:
line 61, should we call RHSM Red Hat Subscription Manager?

Reply:
Fixt